app/retail/emails.py

```diff
@@ -1085,11 +1085,13 @@ def render_new_bounty_roundup(to_email):
         'invert_footer': False,
         'hide_header': False,
         'highlights': highlights,
+        'kudos_friday': kudos_friday,
```

Comment:
I wasn't aware that the kudos is inserted into the intro. I'd understood that only the textual content is what's loaded into the intro. It would be helpful if you could provide a sample email content as it would be in the RoundupEmail model.

Reply:
I shall remove the code that passes and inserts the kudos_friday again into the template.